js/src/workspaces/window.js

```diff
@@ -627,9 +626,17 @@
             }
             return true;
           });
+
+          jQuery.each( _this.annotationsList, function( key, value ) {
+
+            if(jQuery.isArray(value.resource)){
+               _this.annotationsList[key].resource = value.resource[0];
```

Comment:
Nice fix. Now we don't have to worry about these differences, since (I think) they are both valid.